<!-- visualization.launch - FSD 可视化节点 -->
<launch>
    <arg name="enable_cones" default="true"/>
    <arg name="enable_path" default="true"/>
    <arg name="enable_vehicle" default="true"/>
    <arg name="frame_id" default="velodyne"/>
    <arg name="show_trail" default="true"/>
    <arg name="trail_length" default="200"/>

    <node name="fsd_viz_node" pkg="fsd_visualization" type="fsd_viz_node" output="screen">
        <param name="enable_cones" value="$(arg enable_cones)"/>
        <param name="enable_path" value="$(arg enable_path)"/>
        <param name="enable_vehicle" value="$(arg enable_vehicle)"/>
        <param name="frame_id" value="$(arg frame_id)"/>
        <param name="show_trail" value="$(arg show_trail)"/>
        <param name="trail_length" value="$(arg trail_length)"/>
    </node>
</launch>
